src/main/java/ch/njol/skript/tests/platform/PlatformMain.java

```diff
@@ -58,6 +61,9 @@ public static void main(String... args) throws IOException, InterruptedException
 		Path envsRoot = Paths.get(args[3]);
 		assert envsRoot != null;
 		boolean devMode = "true".equals(args[4]);
+		boolean genDocs = false;
+		if (args.length > 5)
+			genDocs = "true".equals(args[5]);
```

Comment:
Why make this parameter optional? It will only make things more annoying if we add more parameters in the future

Reply:
Sure